zerver/tests/test_email_log.py

```diff
@@ -11,7 +11,7 @@ class EmailLogTest(ZulipTestCase):
     def test_generate_and_clear_email_log(self) -> None:
         with self.settings(EMAIL_BACKEND='zproject.email_backends.EmailLogBackEnd'), \
                 mock.patch('zproject.email_backends.EmailLogBackEnd.send_email_smtp'), \
-                mock.patch('logging.info', return_value=None), \
+                self.assertLogs(level="INFO"), \
```

Comment:
We should capture and assert the value of the logged value, like we do with most existing `assertLogs` statements.